addons/base_onboarding/models/base_onboarding.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from .base_onboarding_state import ONBOARDING_STATES
+from odoo.tools.json import scriptsafe as json_safe
+
+
+class Onboarding(models.Model):
+    _name = 'base.onboarding'
+    _description = 'Onboarding'
+
+    name = fields.Char('Name of the onboarding')
+    route_name = fields.Char(
+        'Short name to use in route', default=lambda s: s.name.replace(' ', '') if s.name else 'route')
+    step_ids = fields.One2many('base.onboarding.step', 'onboarding_id', 'Onboarding steps')
+
+    panel_background_color = fields.Selection(
+        [('orange', 'Orange'), ('blue', 'Blue')], default='orange')
+    panel_background_image_url = fields.Char("URL for the panel's background image")
+    panel_properties = fields.Text(
+        'Panel template properties', compute='_compute_panel_properties', store=False, default='')
```

Comment:
store False is the default value, no need to give it explicitly (and no default, as it should always be covered by the compute itself)